src/LinAlg/SystemMatrix.h

```diff
@@ -214,6 +214,9 @@ class SystemMatrix
                               const LinSolParams& spar,
                               LinAlg::LinearSystemType ltype);
 
+  static Type GlbL2Type; //!< Matrix type for global L2 projection
+  static LinSolParams* GlbL2Params; //!< Linear solver params for global L2 projection
```

Comment:
Tror jeg heller ville hatt disse to static variables i GlbL2 scope (dvs. GlbL2projector.h).

Reply:
okay, valgte denne løsningen pga mindre header contamination, men den er grei.